contract_journal/views/contract.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="account_analytic_account_journal_form" model="ir.ui.view">
+        <field name="name">account.analytic.account.journal.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='template_id']" position="before">
```

Comment:
Admittedly I should be testing my 9.0 submissions better, but mannnnn switching over to the v9 instance from the v10 in my local is a real pain 😦 . I swear one of these days I'm going to create us some sort of `odooenv`